common/buildcraft/transport/stripes/StripesHandlerMinecartDestroy.java

```diff
@@ -43,13 +43,13 @@ public boolean handle(World world, BlockPos pos, EnumFacing direction, EntityPla
                         container.setInventorySlotContents(i, StackUtil.EMPTY);
                         // Safety check
                         if (container.getStackInSlot(i).isEmpty()) {
-                            activator.sendItem(s, direction.getOpposite());
+                            activator.sendItem(s, direction);
                         }
                     }
                 }
             }
             cart.setDead();
-            activator.sendItem(StackUtil.asNonNull(cart.getCartItem()), direction.getOpposite());
+            activator.sendItem(StackUtil.asNonNull(cart.getCartItem()), direction);
             return true;
```

Comment:
Unfortunately I can't test this as block breaking is broken... oh well. I'll have to look at that later I suppose.